src/cinder/Log.cpp

```diff
@@ -318,6 +409,44 @@ void LoggerNSLog::write( const Metadata &meta, const string &text )
 	NSLog( @"%@ %@", metaDataNs, textNs );
 }
 
+// ----------------------------------------------------------------------------------------------------
+// MARK: - LoggerNSLog
+// ----------------------------------------------------------------------------------------------------
+
+LoggerSysLog::LoggerSysLog()
+{
+	// Most general approach for determining app name.  Perhaps there is something more cinder specific?
+	// https://developer.apple.com/library/mac/qa/qa1544/_index.html
+
+	NSString *bundlePath = [[NSBundle mainBundle] bundlePath];
+	NSString *appName = [[NSFileManager defaultManager] displayNameAtPath: bundlePath];
+
+	const char *cAppName = [appName UTF8String];
+	openlog( cAppName, ( LOG_CONS | LOG_PID), LOG_USER );
+}
+
+LoggerSysLog::~LoggerSysLog()
+{
+	closelog();
+}
+
+void LoggerSysLog::write( const Metadata &meta, const string &text )
+{
+	// TODO: code duplication
+	int sysLevel;
+	switch ( meta.mLevel ) {
+		case LEVEL_VERBOSE:		sysLevel = LOG_DEBUG;	break;
+		case LEVEL_INFO:		sysLevel = LOG_INFO;	break;
+		case LEVEL_WARNING:		sysLevel = LOG_WARNING;	break;
+		case LEVEL_ERROR:		sysLevel = LOG_ERR;		break;
+
+		// debatable if FATAL is a LOG_CRIT or LOG_ALERT
+		case LEVEL_FATAL:		sysLevel = LOG_CRIT;	break;
+		default: CI_ASSERT_NOT_REACHABLE();
+	}
+	syslog(sysLevel, "%s %s", meta.toString().c_str(), text.c_str());
```

Comment:
I had the same issue until I found some info about /etc/asl.conf configurations.  I tried to comment about this during my PR but I wasn't clear enough.

I'm going to change it so it's always LOG_NOTICE and up.  Part of me doesn't love flooding the syslog with everything.  Chances are if you use the syslog you're linking it to some type of process manager and you really only need high priority messages.